### DISCUSSION TITLE
Esp32 S3 won't show up as accesspoint

### DISCUSSION THREAD

**Discussion by Robbedoes2000** [tbnobody/OpenDTU#1818] [Q&A]
Hi, I recently got a mini ESP32 S3 zero to use as DTU. However, I am not successful in booting it up. I flashed it with both the USB .bin and non USB .bin. It flashes without errors, but then it's just a brick. No WiFi AP visible on my phone. I don't have a second ESP to see if mine is faulty. I reflashed it several times, power cycled it after flashing, waited for half an hour. Still nothing. 

I can confirm the ESP works, as esphome can flash it and it connects to my WiFi.

Any suggestions?

**Comment by MetaChuh** [comment]
hi @Robbedoes2000 

did you flash [`opendtu-generic_esp32s3.factory.bin`](https://github.com/tbnobody/OpenDTU/releases/download/v24.2.16/opendtu-generic_esp32s3.factory.bin) ?
(direct link)

info: the first flash has to be a `factory` flash, otherwise the needed partitioning might not match with the current partitioning of an esp32 (s3)

**Reply by Robbedoes2000** [reply]
Thanks for your reply! Yes, now I did, but still nothing. Also tried an older version - nothing.

**Comment by Robbedoes2000** [comment]
Well, using a Lolin S3 board it is working. So no idea why the zero board from Tenstar isn't working.

**Reply by donotoverdriveit** [reply]
Do you have the PIN assignment for S3 **Mini** + nRF24 for me?
